init.ts

```diff
@@ -287,11 +287,20 @@ await Deno.writeTextFile(
   ROUTES_API_JOKE_TS,
 );
 
-const TWIND_CONFIG_TS = `import { Options } from "$fresh/plugins/twind.ts";
+const TWIND_CONFIG_TS = `import { Options } from "$fresh/plugins/twindv1.ts";
+
+import { Preset } from "https://esm.sh/v131/@twind/core@1.1.3/core.d.ts";
```

Comment:
Would this `import { Preset } from "https://esm.sh/@twind/core@1.1.3";` look more the rest of imports?